articles/other/other-vid-kc-contribution.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Editing Knowledge Centre articles video | UKCloud Ltd
+description: Shows a video that explains how to edit Knowledge Centre articles within GitHub
+services: other
+author: Sue Highmoor
+
+toc_rootlink: Videos
+toc_sub1: 
+toc_sub2:
+toc_sub3:
+toc_sub4:
+toc_title: Editing Knowledge Centre articles video
+toc_fullpath: Videos/other-vid-kc-contribution.md
+toc_mdlink: other-vid-kc-contribution.md
+---
+
+# Editing Knowledge Centre articles video
+
+The following shows you how you can make improvements to Knowledge Centre articles by editing them in GitHub.
```

Comment:
Should this be "The following video" to keep consistency?

Reply:
Indeed it should. Fixed